README.md

```diff
@@ -219,11 +220,24 @@ Since for streaming responses OpenAI doesn't supply the full usage tokens summar
 
 NOTE: Stream is currently only supported with SDK usage, and not with using REST directly.
 
-### Langchain support
+## LangChain support
 
-We currently only support monitoring OpenAI endpoints when using Langchain via monkey-patching. See completion_langchain_monkeypatch.py for an example.
+You can use the exported `monitor_langchain_llm` to wrap a LangChain OpenAI LLM (chat or normal) with Mona's monitoring capabilities.:
 
-A more robust Langchain integration is in the works.
+```py
+from mona_openai import monitor_langchain_llm
+
+from langchain.llms import OpenAI
+
+# Loading Mona credetntials etc...
```

Comment:
Actually, I think we should mention explicitly what we are loading here

Reply:
Changed this completely. I was a mistake here.